panic if struct has unexported value.

If struct has unexported value like this, CreateTable() will panic.

```
type Test struct {
    Exported   string
    unexported string
}
func main() {
    db, _ := GetDBForTest()
    db.Debug().CreateTable(Test{})
}
```

The error is 

```
$ go run main.go
panic: reflect.Value.Interface: cannot return value obtained from unexported field or method
```

This is occurred after 8482d0a121cb5b32ad0773b4742b93b748d15f2f.

--- a/scope_private.go
+++ b/scope_private.go
@@ -5,6 +5,7 @@ import (
 	"database/sql/driver"
 	"errors"
 	"fmt"
+	"go/ast"
 	"reflect"
 	"regexp"
 	"strconv"
@@ -550,6 +551,9 @@ func (scope *Scope) createTable() *Scope {
 	fields := scope.Fields()
 	scopeType := scope.IndirectValue().Type()
 	for i := 0; i < scopeType.NumField(); i++ {
+		if !ast.IsExported(scopeType.Field(i).Name) {
+			continue
+		}
 		for _, field := range scope.fieldFromStruct(scopeType.Field(i), false) {
 			field = fields[field.DBName]
 			if field.IsNormal {

--- a/structs_test.go
+++ b/structs_test.go
@@ -125,12 +125,13 @@ func (i *Num) Scan(src interface{}) error {
 }
 
 type Animal struct {
-	Counter   uint64    `gorm:"primary_key:yes"`
-	Name      string    `sql:"DEFAULT:'galeone'"`
-	From      string    //test reserved sql keyword as field name
-	Age       time.Time `sql:"DEFAULT:current_timestamp"`
-	CreatedAt time.Time
-	UpdatedAt time.Time
+	Counter    uint64    `gorm:"primary_key:yes"`
+	Name       string    `sql:"DEFAULT:'galeone'"`
+	From       string    //test reserved sql keyword as field name
+	Age        time.Time `sql:"DEFAULT:current_timestamp"`
+	unexported string    // unexported value
+	CreatedAt  time.Time
+	UpdatedAt  time.Time
 }
 
 type Post struct {

